apollo-router/src/services/subgraph_service.rs

```diff
@@ -674,6 +702,18 @@ async fn call_http(
     let display_headers = context.contains_key(LOGGING_DISPLAY_HEADERS);
     let display_body = context.contains_key(LOGGING_DISPLAY_BODY);
 
+    let signing_params = {
+        let ctx = context.private_entries.lock();
+        let sp = ctx.get::<SigningParamsConfig>();
+        sp.cloned()
+    };
```

Comment:
NIT
```suggestion
  let signing_params = context.private_entries.lock().get::<SigningParamsConfig>().cloned();
```

Reply:
good catch! fixed in [aca6ce3](https://github.com/apollographql/router/pull/3735/commits/aca6ce3cb4ce2c7abc777fd4e1dca4a09e9c64eb)